editor/header/tools/index.js

```diff
@@ -2,10 +2,10 @@
  * Internal dependencies
  */
 import './style.scss';
-import Dashicon from '../../components/dashicon';
-import IconButton from '../../components/icon-button';
-import Inserter from '../../components/inserter';
-import Button from '../../components/button';
+import Dashicon from '../../../components/dashicon';
+import IconButton from '../../../components/icon-button';
+import Inserter from 'inserter';
+import Button from '../../../components/button';
```

Comment:
Same, we shouldn't need these relative paths any more.  If I'm wrong about that, then it seems like we need to reconsider how the import paths work.